package.json

```diff
@@ -13,7 +13,8 @@
     "build:affected": "yarn nx affected --target=build --parallel $(yarn get:cpus-number) --output-style stream",
     "build:tools": "yarn nx run-many --target=build --projects=eslint-plugin,dev-tools --parallel $(yarn get:cpus-number)",
     "build:swagger-gen": "yarn nx run ama-sdk-generator-sdk:build-swagger",
-    "publish": "yarn nx run-many --target=publish --parallel $(yarn get:cpus-number) --nx-bail",
+    "prepare-publish": "prepare-publish **/!(dist)/dist",
```

Comment:
you could actually have the pattern directly in the script by reading the `workspaces` field from the package.json to avoid having issue with globs that could also target a *templates* folder or other ones

Reply:
done